src/chapters/03_Binomial_coefficients_are_(almost)_never_powers.lean

```diff
@@ -1,18 +1,14 @@
 /-
 Copyright 2022 Google LLC
-
 Licensed under the Apache License, Version 2.0 (the "License");
 you may not use this file except in compliance with the License.
 You may obtain a copy of the License at
-
     https://www.apache.org/licenses/LICENSE-2.0
-
 Unless required by applicable law or agreed to in writing, software
 distributed under the License is distributed on an "AS IS" BASIS,
 WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
-
 Authors: Moritz Firsching, Christopher Schmidt
```

Comment:
Could you leave this haeder as is, that is not remove the newlines

Reply:
Yes, that was by accident